fire/docstrings_test.py

```diff
@@ -165,7 +165,208 @@ def test_google_format_multiline_arg_description(self):
                     description='The first parameter.'),
             ArgInfo(name='param2', type='str',
                     description='The second parameter. This has a lot of text, '
-                                'enough to cover two lines.'),
+                                'enough to\ncover two lines.'),
+        ],
+    )
+    self.assertEqual(expected_docstring_info, docstring_info)
+
+  def test_google_format_long_arg_long_description(self):
+    docstring = """This is a Google-style docstring with long args.
+
+    Args:
+    function_maker_handler_event_factory: The function-maker-handler event factory
+      responsible for making the function-maker-handler event. Use this whenever
+      you need a function-maker-handler event made for you programmatically. 
+    """
+    docstring_info = docstrings.parse(docstring)
+    expected_docstring_info = DocstringInfo(
+        summary='This is a Google-style docstring with long args.',
+        args=[
+            ArgInfo(name='function_maker_handler_event_factory',
+                    description='The function-maker-handler event factory '
+        'responsible for making the function-maker-handler event. '
+        'Use this whenever\nyou need a function-maker-handler event'
+         ' made for you programmatically.'),
+        ],
+    )
```

Comment:
It may be good to add an example here, or below, which contains a second line in the documentation before the args (since this is very common). e.g. 
```suggestion
    docstring = """This is a Google-style docstring with long args.
    
    Here is some further and potentially long-winded explanation of the function.
    
    Args:
        function_maker_handler_event_factory: The function-maker-handler event factory
            responsible for making the function-maker-handler event. Use this whenever
            you need a function-maker-handler event made for you programmatically. 
    """
    docstring_info = docstrings.parse(docstring)
    expected_docstring_info = DocstringInfo(
        summary='This is a Google-style docstring with long args.\n\nHere is some further and potentially long-winded explanation of the function.',
        args=[
            ArgInfo(name='function_maker_handler_event_factory',
                    description='The function-maker-handler event factory '
        'responsible for making the function-maker-handler event. '
        'Use this whenever\nyou need a function-maker-handler event'
         ' made for you programmatically.'),
        ],
    )
```

Reply:
I like it. I think this test case would fail rn since the default behavior is to ignore the \n and just display without them. 

This was the issue I fixed but I think I have fixed it within the scope of args but not the name of the function and summary. I will work on this as well.

Reply:
Another thing I noticed was that the name of the function is displayed as <name> -  <summary>. Is that how it should be? I only ask since we have a section called description where we display the summary.

Reply:
> Another thing I noticed was that the name of the function is displayed as <name> -  <summary>.

I think so, though maybe there are examples where it doesn't make sense and we can improve it. I'd have to give that some thought, but fortunately that's orthogonal to this change.